compiler/ballerina-compiler-api/src/main/java/org/ballerina/compiler/api/symbols/Symbol.java

```diff
@@ -44,17 +45,25 @@
 
     /**
      * Get the Symbol Kind.
-     * 
+     *
      * @return {@link SymbolKind} of the symbol
      */
     SymbolKind kind();
 
     /**
      * Get the Documentation attachment bound to the symbol.
-     * 
+     *
      * @return {@link Optional} doc attachment
      */
     Optional<Documentation> docAttachment();
 
+    /**
+     * This retrieves position information of the symbol in the source code. The position given here is the location of
+     * the definition of the symbol.
+     *
+     * @return The position information of the symbol
+     */
+    Location position();
```

Comment:
Went with `position()` for this since this is the term we are used to in the compiler. Should this be changed to something else? `location()` for example?

Reply:
What if we change the model name Position? Either way I would +1 for `Position position()` or `Location location()`

Reply:
+1 for `location()`.
We use the same in the syntax tree as well.